app/gui/view/graph-editor/src/lib.rs

```diff
@@ -450,7 +450,8 @@ ensogl::define_endpoints_2! {
 
 
         // === Layout ===
-        space_for_window_buttons (Vector2<f32>),
+
+        top_bar_offset_x (f32),
```

Comment:
`top_bar_offset_x` roughly maintains the same function as `space_for_window_buttons`, but I gave it a more accurate name, since (1) it's no longer just an offset for window buttons but also the dashboard button, (2) only the offset in the x-axis was used and (3) it's not within the responsibilities of the graph editor to know what sort of items are placed in the top bar by the project view.

Reply:
The old name is just not true anymore since it's not just space for window buttons. If you can think of a better name, feel free to share it.